### ISSUE TITLE
Command doesn't add changes to the staging area

### ISSUE THREAD

**Issue by raisedadead** [rcmdnk/homebrew-file#164]
Running `brew file push` doesn't add the changes to the staging area. I discovered this first when I was running the `brew file update` command).

If it helps, I do not use the `wrap` feature. 

Here is 

```
brew file push
$ git add -A
$ git commit -m "Update the package list"
On branch main
Your branch is up to date with 'origin/main'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   Brewfile
no changes added to commit (use "git add" and/or "git commit -a")
$ git push
Everything up-to-date
```

**Comment by rcmdnk** [comment]
Thanks for the report!
The bug was embedded in v9.0.0.

It should be fixed at v9.0.10 (fo both `brew file push` and `brew file update`).
Could you please try the updated one?

**Comment by raisedadead** [comment]
I just tested it, works like a charm. Thanks for fixing this.
